src/main/scala/com/gu/zuora/model/response/SubscriptionsResponse.scala

```diff
@@ -19,12 +19,7 @@ object RatePlan {
 
 case class SubscriptionsResponse(subscriptions: List[Subscription])
 
-case class Subscription(accountNumber: String, status: String, ratePlans: List[RatePlan]) {
-  def isActive: Boolean = status == "Active"
-  def hasContributorPlan(config: ZuoraConfig, billingPeriod: BillingPeriod): Boolean = {
-    ratePlans.exists(_.productRatePlanId == config.contributionConfig(billingPeriod).productRatePlanId)
-  }
-}
+case class Subscription(accountNumber: String, status: String, ratePlans: List[RatePlan])
```

Comment:
I have made a DomainSubscription that does all these defs in the conversion, so no need the internal methods now.